### ISSUE TITLE
Windows 10: Alt-Printscreen on a windowed game exits the game.

### ISSUE THREAD

**Issue by GraionDilach** [Pyrdacor/freeserf.net#118]
`freeserf.log` contains:

```
11:30:42 Error: [Application] KeyDown: Index was outside the bounds of the array.
11:30:42 Error: [Application] Exception: Index was outside the bounds of the array.
```

**Comment by Pyrdacor** [comment]
Fixed it in code. Next release will contain the change.

**Comment by Pyrdacor** [comment]
Should be fixed in v2.0.8.

**Comment by GraionDilach** [comment]
Confirmed.
